src/app/files.js

```diff
@@ -2,12 +2,14 @@
 
 var EventManager = require('../lib/eventManager')
 
+var FILENAME = 'Untitled'
+
 function Files (storage) {
   var event = new EventManager()
   this.event = event
   var readonly = {}
 
-  this.exists = function (path) {
+  this.exists = function isFile (path) {
```

Comment:
oh this helped in the debugger to show not lots of anonymous functions but named ones.